packages/flutter/lib/src/widgets/raw_keyboard_listener.dart

```diff
@@ -46,6 +46,8 @@ class RawKeyboardListener extends StatefulWidget {
        assert(child != null),
        super(key: key);
 
+  /// Controls whether this widget has keyboard focus.
```

Comment:
The macro below contains the documentation. In the docs it looks like this: https://master-api.flutter.dev/flutter/widgets/RawKeyboardListener/focusNode.html

This change doesn't seem to add more meaningful documentation than what's already there?

Reply:
The changes is to increase clarity when reading the documentation from https://api.flutter.dev/flutter/widgets/RawKeyboardListener-class.html according to the issue reported from #41475

Reply:
Even with this change the documentation for the property will still say that the focus node is optional because of the macro even though its marked as required in the constructor. 

/cc @gspencergoog

Reply:
@goderbauer , I've removed the macro to make the documentation less confusing.

Reply:
We make use of macros for documenting classes in many places throughout the code base. @kumabotz, do you find them confusing in general or just in this specific case?

Reply:
My understanding is that the macro is wrong. The macro claims this property is optional and talks about what it means to set or not set it, but the constructor has this property actually marked as required. Not setting it is not an option.